src/components/bar.cpp

```diff
@@ -220,6 +220,8 @@ bar::bar(connection& conn, signal_emitter& emitter, const config& config, const
 
   m_opts.struts = m_conf.get(bs, "enable-struts", m_opts.struts);
 
+  m_opts.disable_hover_checking = m_conf.get(bs, "disable-hover-checking", m_opts.disable_hover_checking);
```

Comment:
For consistency we should name this `enable-hover-checking` or just `enable-hover`. The code exclusively uses the inverse of this option anyway.

We should maybe also disable it by default to not constantly run the mouse move events when it's not used.

Reply:
Okay! My thought was that the comfort of not having to explicitly enable a feature that most people would probably use beats the performance. So the option of disabling would be just for people who worry about performance and aren't using it.

But I guess for now that no module actually makes use of it, it makes sense to have that behavior and maybe in the future if lots of modules use it and having to manually enable it becomes cumbersome it could be changed back to how I imagined it.

Reply:
Exactly, we can always turn it on by default later (but turning it off by default would break many people's configs that rely on this).